front/app/modules/commercial/content_builder/admin/components/ContentBuilderLayout/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { colors, media, stylingConsts } from 'utils/styleUtils';
+import useFeatureFlag from 'hooks/useFeatureFlag';
+import { withRouter, WithRouterProps } from 'react-router';
+import { RightColumn } from 'containers/Admin';
+import { Editor } from '@craftjs/core';
+import { Box } from '@citizenlab/cl2-component-library';
+import ContentBuilderToolbox from '../ContentBuilderToolbox';
+import ContentBuilderSettings from '../ContentBuilderSettings';
+import Container from '../CraftComponents/Container';
+import RenderNode from '../RenderNode';
+
+const Wrapper = styled.div`
+  flex: 0 0 auto;
+  width: 210px;
+`;
+
+const ContainerInner = styled.nav`
+  flex: 0 0 auto;
+  width: 210px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  padding-top: ${stylingConsts.menuHeight + 10}px;
+  background-color: ${colors.disabledPrimaryButtonBg};
+  border-right: 1px solid ${colors.border};
+
+  ${media.smallerThan1200px`
+    width: 80px;
+  `}
+`;
+
+type ContentBuilderLayoutProps = {
+  onMount: (isVisible: boolean) => void;
+} & WithRouterProps;
+
+const ContentBuilderLayout: React.FC<ContentBuilderLayoutProps> = ({
```

Comment:
We are creating the new layout by reusing some admin styles to keep it consistent with the rest of the admin pages.